pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -49,7 +49,10 @@ self: super: {
   Dust-crypto = dontCheck super.Dust-crypto;
   hasql-postgres = dontCheck super.hasql-postgres;
   hspec = super.hspec.override { stringbuilder = dontCheck self.stringbuilder; };
-  hspec-core = super.hspec-core.override { silently = dontCheck super.silently; temporary = dontCheck super.temporary; };
+  hspec-core = dontCheck (super.hspec-core.override { silently = dontCheck self.silently; });
```

Comment:
This change doesn't look right. The original version did run the `hspec-core` test suite and it needed special untested variants of some dependencies to accomplish that without creating an infinite loop during evaluation. The new code does not run test suite, however, and therefore it won't need the untested version of `silently`. I would suggest to either (a) enable tests of `hspec-core` (which is what I'd prefer) or (b) to remove the override that passes a special version of `silently`.